packages/paste-core/components/form-pill-group/src/FormPillGroup.tsx

```diff
@@ -41,12 +44,18 @@ FormPillGroupStyles.displayName = 'StyledFormPillGroup';
  * </FormPillGroup>
  * @see http://paste.twilio.design/components/form-pill-group
  */
-export const FormPillGroup = React.forwardRef<HTMLUListElement, FormPillGroupProps>((props, ref) => {
-  return (
-    <Composite as={FormPillGroupStyles} {...props} ref={ref}>
-      {props.children}
-    </Composite>
-  );
-});
+export const FormPillGroup = React.forwardRef<HTMLUListElement, FormPillGroupProps>(
+  ({i18nKeyboardControls = 'Press Delete or Backspace to remove. Press Enter to toggle selection.', ...props}, ref) => {
+    const keyboardControlsId = useUID();
+    return (
+      <>
+        <Composite as={FormPillGroupStyles} {...props} ref={ref} aria-describedby={keyboardControlsId}>
+          {props.children}
+        </Composite>
+        <ScreenReaderOnly id={keyboardControlsId}>{i18nKeyboardControls}</ScreenReaderOnly>
+      </>
+    );
+  }
+);
 
```

Comment:
question: is there not a propTypes section for Form Pill Group?

Reply:
🤷🏽 it doesnt.... i've noticed that not all components use prop types. I can bring this up as a 16th min question